MangoPay/PayIn.php

```diff
@@ -43,6 +60,32 @@ class PayIn extends Transaction
      */
     public $RecurringPayinRegistrationId;
 
+    /**
+     * The IP address of the end user initiating the transaction, in IPV4 or IPV6 format.
+     * @var string
+     */
+    public $IpAddress;
+
+    /**
+     * Information about the browser used by the end user (author) to perform the payment.
+     * @var BrowserInfo
+     */
+    public $BrowserInfo;
```

Comment:
What is the reason for adding this to the PayIn class?

Reply:
This property was identified in both the documentation and PHP SDK. I verified its existence through debug logs and implemented it accordingly.